andela/travel_tool_front
# #161840270 Integrate date range filter

#### What does this PR do?
Integrate date-range-picker calender

#### Description of Task to be completed?
- Integrate  date-range-picker calender
- Match displayed data with the corresponding titles
- Match the current location with the displayed data
- Implement download csv actions

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
```ft-date-range-filter-161840270```
- Install dependencies:
```yarn install```
`yarn test`
- All the tests should be passing
- Start the application:
```yarn start```

- Login to the application
- Navigate to the dashboard page on the sidebar
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
1. To use the Dashboard - Create your standalone component and plug it into the dashboard component.
2. You should have the user role of a Travel Administrator or Super Administrator to be able to see the `manage` page. To do that:
  - Download `Postico` and connect to your travela database with relevant details: Db name, password, port, user
  - Click `userRoles` table and set the `roleId` to that of a Travel Administrator(29187) or Super[]Administrator(10948) 

#### What are the relevant pivotal tracker stories?
[#161840270](https://www.pivotaltracker.com/story/show/161840270)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-11-09 at 16 49 04" src="https://user-images.githubusercontent.com/22371596/48266114-7a41b800-e43f-11e8-94e0-a3a02d427119.png">
